chart/README.md

```diff
@@ -32,17 +32,18 @@ This repository is intended to store a helm chart to create a cluster of Nexus I
 ## Running
 
 1. Start your Kubernetes cluster if needed
-2. Open a console/terminal in the helm chart directory
+2. Open a console/terminal
 3. Switch to the correct context to use your cluster if needed (e.g. `kubectl config use-context my-context`)
-4. Install the helm chart dependencies via
-   `helm dependency update .`
+4. Add the helm chart repository via
+   `helm repo add sonatype https://sonatype.github.io/helm3-charts/`
 5. Install the helm chart via
-   `helm install --namespace <namespace> <name> <overrides> .`.
+   `helm install --namespace <namespace> <name> --dependency-update <overrides> sonatype/nexus-iq-server-ha --version <version>`.
 where
-   1. `<name>` can be any name for the helm chart
-   2. `<namespace>` can be an existing namespace for the helm chart (created prior via 
-   `kubectl create namespace <namespace>`, or to create automatically include the flag `--create-namespace`)
+   1. `<namespace>` can be an existing namespace for the helm chart (created prior via
+      `kubectl create namespace <namespace>`, or to create automatically include the flag `--create-namespace`)
+   2. `<name>` can be any name for the helm chart
    3. `<overrides>` is a set of overrides for values in the helm chart (see below)
+   4. `<version>` is the version of the helm chart to use (e.g. `154.0.0`)
```

Comment:
I would remove the version example here because it might be taken as a recommendation